src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcDsa.cs

```diff
@@ -25,5 +28,49 @@ internal static partial class Crypto
         // returns the maximum length of a DER encoded ECDSA signature created with this key.
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcDsaSize")]
         internal static extern int EcDsaSize(SafeEcKeyHandle ecKey);
+
+        // todo: move this to common cng and openssl
```

Comment:
It was referring to a common wrapper for obtaining a valid Oid by first using the value and if that fails use the friendly name. It should do this without throwing an exception which rules out the current Oid implementation as that throws if not found. It also throws a crypto exception instead of platformnotsupported exception. For now, I removed the comment and am tracking this separately